src/modules/class.js

```diff
@@ -0,0 +1,32 @@
+import {addToLocalStorage} from "./localstorage.js"
+
+class Task {
```

Comment:
- [ ] Please you have issues with your codes. Kindly fix it. 
- [ ] Make sure all the elements of the user interface will be fully functional according to the Project requirements.
![Capture](https://user-images.githubusercontent.com/63972670/194655383-4fc3cf6c-63f5-44e3-8b05-d7726fd0974f.JPG)